pha-admin/src/main/java/com/store/pharmacy/bizpartner/service/impl/BizPartnerServiceImpl.java

```diff
@@ -0,0 +1,192 @@
+			bizPartner.setBizPartnerName(bizPartnerInput.getBizPartnerName());
+			bizPartner.setTaxCode((bizPartnerInput.getTaxCode()));
+			bizPartner.setBizTypeCode(bizPartnerInput.getBizTypeCode());
+			bizPartner.setCountry(bizPartnerInput.getCountry());
+			bizPartner.setAddress(bizPartnerInput.getAddress());
+			bizPartner.setEmail(bizPartnerInput.getEmail());
+			bizPartner.setPhoneNumber(bizPartnerInput.getPhoneNumber());
+			bizPartner.setDescription(bizPartnerInput.getDescription());
+			bizPartner.setCreatedBy(bizPartnerInput.getUserId());
+			bizPartner.setCreatedAt(today);
+			bizPartner.setStatus(bizPartnerInput.getStatus());
+			bizPartner.setEnabled(bizPartnerInput.getEnabled());
+		}
+		return bizPartner;
+
+	}
+
+	public BizPartnerOutput convertBizPartnerToBizPartnerOutput(BizPartner bizPartner,  String action){
+		BizPartnerOutput bizPartnerOutput = new BizPartnerOutput();
+
+		bizPartnerOutput.setBizPartnerId(bizPartner.getBizPartnerId());
+		bizPartnerOutput.setBizPartnerName(bizPartner.getBizPartnerName());
+		bizPartnerOutput.setTaxCode(bizPartner.getTaxCode());
+		bizPartnerOutput.setBizTypeCode(bizPartner.getBizTypeCode());
+		bizPartnerOutput.setCountry(bizPartner.getCountry());
+		bizPartnerOutput.setAddress(bizPartner.getAddress());
+		bizPartnerOutput.setEmail(bizPartner.getEmail());
+		bizPartnerOutput.setPhoneNumber(bizPartner.getPhoneNumber());
+		bizPartnerOutput.setDescription(bizPartner.getDescription());
+		bizPartnerOutput.setEnabled(bizPartner.getEnabled());
+
+		if (action.equals(INSERT)){
+			bizPartnerOutput.setUpdateBy(bizPartner.getCreatedBy());
+			bizPartnerOutput.setUpdateAt(bizPartner.getUpdatedAt());
+		}
+		else if (action.equals(UPDATE) || action.equals(SEARCH) ){
+			bizPartnerOutput.setUpdateBy(bizPartner.getUpdatedBy());
+			bizPartnerOutput.setUpdateAt(bizPartner.getUpdatedAt());
+		}
+		return  bizPartnerOutput;
+	}
+
+	/**
+	 *
+	 * @param input
+	 * @return
+	 */
+	public String getBizTypeCodeName (String input){
+		Common common =  commonRepository.findByCommonCodeAndName(BizPartnerConstant.BIZ_TYPE_CODE, input.trim());
+		if (common == null)
+			return "";
+		return common.getName();
+	}
+
+	/**
+	 *
+	 * @param input
+	 * @return
+	 */
+	public String getStatus (String input){
+		Common common =  commonRepository.findByCommonCodeAndName(BizPartnerConstant.STATUS, input.trim());
+		if (common == null)
+			return "";
+		return common.getName();
+	}
+	/**
+	 *
+	 * @param input
+	 * @return
+	 */
+	public boolean checkBizTypeCode (String input ){
+		if (!input.equals(getBizTypeCodeName(input))  )
+			throw new DataNotFoundException("MSG0025" ,new Object[] {labelService.findByExactLabelCodes("C006","en")});
+		return true;
+	}
+
+	/**
+	 *
+	 * @param input
+	 * @return
+	 */
+	public boolean checkStatus (String input ){
+		if (!input.equals(getStatus(input))  )
+			throw new DataNotFoundException("MSG0025" ,new Object[] {labelService.findByExactLabelCodes("C007","en")});
+		return true;
+	}
```

Comment:
Hi @vutch308 ,
I think the `checkStatus` function is redundant. Because I cannot see this function used anywhere.
Please help me check it again?

Reply:
Thanks @NguyenMinhThien97 , I will remove this.